### ISSUE TITLE
error when generate a commit message

### ISSUE THREAD

**Issue by rafarosa92** [ai-genie/chatgpt-vscode#148]
### Verify it's not a duplicate bug report

- [X] I searched for similar bugs reported on GitHub and nobody has reported a similar issue. <a href="https://github.com/ai-genie/vscode-chatgpt/issues?q=" target="_blank">Search all issuees</a>.
- [X] I read <a href="https://github.com/ai-genie/vscode-chatgpt#troubleshooting" target="_blank">Troubleshooting guide on Readme</a> and there is not a guide on how to solve my problem.

### Describe the Bug

When I use the command to generate a commit message the error "An error occurred. a.diff is not a function" is thrown
![image](https://github.com/ai-genie/chatgpt-vscode/assets/19498717/99d52775-f7ef-4cbd-a4d1-ce0e7edb7bd0)

### Please tell us if you have customized any of the extension settings or whether you are using the defaults.

I'm using model gpt-4-1106-preview

**Comment by sannwao** [comment]
This bug seems to happen when the github copilot extension is installed.

**Comment by shsh26** [comment]
I haven't installed the github copilot extension, but the same error occurs.

**Comment by liukun-lk** [comment]
When I upgraded to the latest version of VS Code, this issue appeared. The current version of VS Code is 1.86.0.

**Comment by jb19901234** [comment]
> When I upgraded to the latest version of VS Code, this issue appeared. The current version of VS Code is 1.86.0.

Same here

**Comment by marcopausini** [comment]
I have the same issue

**Comment by Xanph** [comment]
This is still an issue. Are there any further suggestions? My VS Code version is 1.86.1. This issue seemed to have happened when OpenAI released their new gpt-4-0125-preview model

**Comment by ogiste** [comment]
Same error here, 

VScode Details - 
Version: 1.86.1
Node.js: 18.17.1
OS: Linux x64 6.5.0-15-generic snap

**Comment by Sigmanor** [comment]
Same error here

Version: 1.86.1
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Linux x64 6.7.4-arch1-1

**Comment by moryw** [comment]
Bump, exact issue I'm getting. Additionally the Lightning Bolt in the commit message box is also missing. I did notice that there is another Generate Commit Message (Experimental) option in the menu which asks for my OpenAI api key which I don't remember being there before. I also did not install CoPilot.



```
Version: 1.86.2
Electron: 27.2.3
ElectronBuildId: 26908389
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Linux x64 6.5.0-18-generic
```

**Comment by DummerYou** [comment]
You can temporarily use the command Ctrl+Shift+P to input "Genie: Generate a commit message" to get the commit message, and wait for the updated version.

**Comment by genieai-info** [comment]
> I think this thread should contain the _exact_ error message (with the misspelling), so people searching for the error actually get directed here: `An error occured. a.diff is not a function`

This should now be fixed in v0.0.11 Please update your extension.

Visual Studio Code has disabled access to quick action button in the textbox. Instead you could use:

1. `ctrl+shift+p` and run `Genie: Generate a commit message` command
2. Or use the menu item under the source control like this:

![image](https://github.com/ai-genie/chatgpt-vscode/assets/128560534/67544254-3fd4-433f-975f-4e1fac6b9a02)
